pillan/cnpg/cnpg-backup.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+cat << END | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cnpg-backup-secrets
+  namespace: cloudnativepg
+type: Opaque
+stringData:
+    ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+    SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+    password: ${SUPERUSER_PASSWORD}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cnpg-backups
+  namespace: cloudnativepg
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "0 12,21 * * *" #8AM CLT - 5PM CLT
+  jobTemplate:
+    spec:
+      template:
+        spec:
```

Comment:
Please set `activeDeadlineSeconds` to a value less than the planned interval between jobs.  This is to prevent a hung job from blocking future backups from running.

Reply:
great fail safe for a cronjob, thanks!